src/settings/watchers/FontWatcher.ts

```diff
@@ -20,9 +20,12 @@ import IWatcher from "./Watcher";
 import { toPx } from '../../utils/units';
 import { Action } from '../../dispatcher/actions';
 import { SettingLevel } from "../SettingLevel";
+import { UpdateSystemFontPayload } from "../../dispatcher/payloads/UpdateSystemFontPayload";
+import { ActionPayload } from "../../dispatcher/payloads";
 
 export class FontWatcher implements IWatcher {
```

Comment:
Some tests for the new functionality on this class might be possible. Desirable if possible.